openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TableProfiler/ColumnProfileTable/ColumnProfileTable.tsx

```diff
@@ -397,9 +397,7 @@ const ColumnProfileTable = () => {
 
   const fetchTableColumnWithProfiler = useCallback(
     async (page: number, searchText: string) => {
-      const tableFQN = tableProfiler?.fullyQualifiedName;
-
-      if (!tableFQN) {
+      if (!fqn) {
```

Comment:
@ShaileshParmar11 I think we need to extract tableFQN since sometimes we have column FQN in the URL as well

Reply:
What if we have it typed in url?

Reply:
Sure, will add.